dotnet-efcore/DynamicDbContextSwitching/DynamicDbContextSwitching/DataSourceProvider.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.Extensions.Configuration;
+
+namespace DynamicDbContextSwitching;
+
+public class DataSourceProvider : IDataSourceProvider
+{
+    public DataSource CurrentDataSource { get; set; }
+
+    private readonly IConfiguration _configuration;
```

Comment:
The order here is not correct, private fields should always appear before the public properties

Reply:
reordered them